.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "main", "master" ]
+  schedule:
+    - cron:  '*/15 * * * *'
+  workflow_dispatch:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          ref: "hp-pages"
+          path: "pages"
+      - uses: actions/checkout@v3
+        with:
+          repository: "SerenityOS/serenity"
+          path: "serenity"
+          fetch-depth: "0"
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+          cache: 'pip' # caching pip dependencies
+      - run: python3 -m pip install -r requirements.txt
+      - name: Install gnuplot
+        run: sudo apt-get install -y gnuplot-nox
+      - name: Generate web page
+        run: |
+          cp pages/*.json .
```

Comment:
Copying *all* json files over feels weird. Note that this seems to be necessary in order to preserve `cache.json` across runs. Again, you get this for free simply by not running on a container.

Reply:
ok. The unfortunate thing is that you are the only one having this knowledge.

Reply:
You are the one who wrote that line, so I'm not sure what you mean?